torch/nn/parallel/distributed.py

```diff
@@ -925,8 +925,15 @@ def forward(self, *inputs, **kwargs):
                 self._check_global_requires_backward_grad_sync(is_joined_rank=False)
 
             if self.device_ids:
+                isLazy = inputs[0].device.type == 'lazy'
```

Comment:
I'm trying to think of a better way to do this, as it is not ideal that lazy tensor details are explicitly encoded in DDP's implementation. 

As a first step, can we define a handler for this logic somewhere in lazy tensor codebase, and import and run this handler in DDP? That will at least abstract out the implementation a bit.

Reply:
Yea, this is definitely not ideal. Maybe I can make `to_kwargs` more robust to identify lazy devices.